rsts/introduction/roadmap.rst

```diff
@@ -0,0 +1,43 @@
+.. _introduction-roadmap:
+
+###############
+Roadmap
+###############
+
+How the core team works
+========================
+So Flyte is used actively in production at Lyft and its various subsidiaries. The core team is very customer focused and cares deeply about a high quality customer experience. Thus we always
+prioritize stability, reliability, observability and maintainability over raw feature development. Features are developed usually in response to specific usecases and user scenarios. That being said,
+we are proactively thinking about the evolution of the system and how we want to keep adapting to changing requirements. Thus most of our changes try to think about future development scenarios and in
+cases where we feel rapid prototyping would enable us to discover potential pitfalls or uncover hidden usecases we proactively developing some features, behind feature flags.
+
+We want to extend the same sense of customer obsession to our Open source community. We would love to hear use-cases that various teams are solving and see how we could adapt parts of Flyte to meet
+those requirements. We welcome collaboration and contributions, but please follow our Contribution Guidelines.
+
+We will divide our roadmap into the multiple sections. Please refer to each sections description to understand what the goal is.
+
+Continuous work
+=================
+This section highlights features and issues that are a continuous area of Focus and will always be in progress. 
+
+1. Documentation 
+----------------
+   Flyte's AIM is to make it absolutely easy for any new user, administrator, contributor to read the documentation and complete their job successfully. We realize That we are not even close. This
+   effort will continue on an ongoing basis and we would love feedback on the top areas where we should improve.
+
+2. Bug Fixes & Reliability Improvements
+----------------------------------------   
+   As mentioned since Flyte is being actively used at Lyft and our scale is always increasing, there are always bugs that we want to squish. So Bugs, are always work in progress and most often then
```

Comment:
This makes it feel like bugs are a low priority, which isn't always the case depending on the severity of the bug.

Reply:
really, I feel this says bugs are found and fixed continuously, but maybe we need to create a process of ranking. I do not want to say we will get to them ASAP, as we have not done that